src/main/java/com/iexec/core/replicate/ReplicatesService.java

```diff
@@ -227,6 +227,7 @@ private boolean isStatusBeforeWorkerLostEqualsTo(Replicate replicate, ReplicateS
     /*
      * This implicitly sets the modifier to POOL_MANAGER
      */
+    @Retryable(value = {OptimisticLockingFailureException.class}, maxAttempts = 100)
```

Comment:
Why do we need a second retry when we have [this](https://github.com/iExecBlockchainComputing/iexec-core/blob/892bce156bc7698ed00fe868c35e12284b6bb08e/src/main/java/com/iexec/core/replicate/ReplicatesService.java#L260) one?

Reply:
Done! :)